drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/data_warehouse_upload_job.rb

```diff
@@ -240,6 +243,23 @@ def case_note_export
       uploader.run!
     end
 
+    def waitlist_events_export
```

Comment:
note, there's an opportunity to DRY up repeated code here. Leaving it as-is for now